src/main/java/com/zombiecastlerush/building/Room.java

```diff
@@ -14,6 +14,7 @@
     public Room(String name, String description) {
         setName(name);
         setDescription(description);
+        setInventory(new Inventory());
```

Comment:
Why do you need a `setInventory` method if you've already initialized a new Inventory in the class variable?

Reply:
removed this. used it in my branch before I got updated branch from @kaashis